packages/bpk-docs/src/pages/InfiniteScrollPage/InfiniteScrollPage.scss

```diff
@@ -0,0 +1,50 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins';
+
+.bpk-docs-infinite-scroll-page {
+  &__fixed-panel {
+    height: 300px; /* stylelint-disable-line unit-blacklist */
+    overflow-y: scroll;
+  }
+
+  &__list {
+    div {
+      height: 4rem;
```

Comment:
We should use a token here, just take one of the existing ones and multiply it to make something of a similar height.